x-pack/plugins/apm/common/__snapshots__/elasticsearch_fieldnames.test.ts.snap

```diff
@@ -220,9 +220,9 @@ exports[`Span METRIC_SYSTEM_FREE_MEMORY 1`] = `undefined`;
 
 exports[`Span METRIC_SYSTEM_TOTAL_MEMORY 1`] = `undefined`;
 
-exports[`Span OBSERVER_LISTENING 1`] = `undefined`;
+exports[`Span OBSERVER_HOSTNAME 1`] = `undefined`;
```

Comment:
NIT: Is it OK for this to be now `undefined`?
Mind that LISTENING and ~NAME~ HOSTNAME have been swapped, but ~NAME~ HOSTNAME is now returning `undefined` in the tests

Reply:
Yeah, that's ok. `hostname` is a different field, and it's not in our test data.